app/views/layout.pug

```diff
@@ -0,0 +1,8 @@
+doctype html
+html
+  head
+    title= title
+    script(src='http://www.chaijs.com/chai.js' type='text/javascript')
+    link(rel='stylesheet', href='/stylesheets/style.css')
+  body
```

Comment:
Personal comment: Why do you like jade (pug). Its so.... white space dependent 🤕